starwars/src/components/StarWarsList.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import StarWarsData from './StarWarsData';
+import 'StarWars.css';
+
+function StarWarsList (props) {
+    return(
+        <div>
+            {props.StarWarsData.map(list => 
+                <StarWarsData key={list.name} list={list} />
+            )}
+        </div>
+    )
+}
```

Comment:
This is your first functional component in your hierarchical file structure, the props that is being sent to this component, you had named "Makers" , again i would consider renaming that to "starwarsChars", so in turn when you map, you would want: props.starwarsChars.map
if you look up the map method in mdn: the syntax for a map function is as follows: 
array.map(function callback){return element for new_array},
you have named the callback function "list", remember this can be anything, "foobar", but for this project
i would consider something like "charsInfo" you have used es6 arrow syntax so it will return the next line,
but you have to keep with the naming convention so change the key and you will be passing:
charsInfo ={charsInfo}